src/ODEs/ODETools/RungeKutta.jl

```diff
@@ -213,12 +214,24 @@ function zero_initial_guess(op::RungeKuttaNonlinearOperator)
   x0
 end
 
-function rhs!(op::RungeKuttaNonlinearOperator, x::AbstractVector)
-  u = x
+function rhs!(op::RungeKuttaStageNonlinearOperator, x::AbstractVector)
+  u = zeros(eltype(x),length(x))
```

Comment:
We cannot do this, we are allocating a global vector here, it should go to the cache

Reply:
clean version does not edit RungeKutta.jl